src/i18n/i18n.js

```diff
@@ -42,19 +42,39 @@ class I18n extends EventHandler {
     }
 
     /**
+     * @private
      * @static
      * @function
      * @name I18n.findAvailableLocale
      * @description Returns the first available locale based on the desired locale specified. First
      * tries to find the desired locale and then tries to find an alternative locale based on the language.
-     * @param {string} desiredLocale - The desired locale e.g. En-US.
+     * @param {string} desiredLocale - The desired locale e.g. en-US.
      * @param {object} availableLocales - A dictionary where each key is an available locale.
      * @returns {string} The locale found or if no locale is available returns the default en-US locale.
+     * @example
+     * // With a defined dictionary of locales
+     * var availableLocales = { en: 'en-US', fr: 'fr-FR' };
+     * var locale = pc.I18n.getText('en-US', availableLocales);
+     * // returns 'en'
      */
     static findAvailableLocale(desiredLocale, availableLocales) {
         return findAvailableLocale(desiredLocale, availableLocales);
     }
 
+    /**
+     * @function
+     * @name I18n#findAvailableLocale
+     * @description Returns the first available locale based on the desired locale specified. First
+     * tries to find the desired locale and then tries to find an alternative locale based on the language.
+     * @param {string} desiredLocale - The desired locale e.g. en-US.
+     * @returns {string} The locale found or if no locale is available returns the default en-US locale.
+     * @example
+     * var locale = this.app.i18n.getText('en-US');
+     */
+    findAvailableLocale(desiredLocale) {
+        return findAvailableLocale(desiredLocale, this._translations);
```

Comment:
so when you call `findAvailableLocale` which function ends up getting executed? I think you should call it as 
`I18n.findAvailableLocale` as that's how you call a static function.

Reply:
ah, ignore me .. the module exports global function of this name and that's what is getting called.